.travis.yml

```diff
@@ -11,3 +11,13 @@ script:
     npm test -- --coverage
 after_success:
   - cat ./coverage/lcov.info | coveralls
+deploy:
+  provider: npm
+  email: jmartine@redhat.com
+  api_key:
+    secure: X0tQXL0stFL2cE+PHM6ZoqDHSgHQEQPJ/uD5FZJanueZ/w9Q2QxVYg9fYj3ITBKZX6DEd95Qsnfx8rwnMf8LJG+snDPhv0oCUWuZAgtaz3W59teAA195nNTkWWrEnIiR1eWZo/apxImSHfdJmjdvhcMYHDAidalNVMOwcTNgDvwmAC55S3CI9mhVqE/7zTJHTWyz4p+tWYlHJTzMTNQMfKrcbYzT9pyWUpKcgRtFDOBdq5Vp/AJahT3NajU4G+UoVOIzq8iDB+q81fnIF8TbV/RrUFB5fLHkYGBo4bw0UOoM2kH7og9Xe6T0GOgBT84bCYQ+V+xkCJIPs/d8/w1dm5fBtzJKsHjFHMDEan2oDNEAN1B1o7sZ0mpVpbBc1/caYLYXJ6GO8L2VpnXFHBOPzOEWi7HPLL5ydn0KQHUR9YeX2jyFZmlhXd7+xlRSHqZIPC0zGQ8JStvz9z5lx02/Oq/1fmRLPf1i8dWmAJkIgTzFa2FRCVhzQZ/uHD64paF81mcOeVq6y/swcoYvj+G+CNyBnAMn1Cgu6eelsHtnr3TzwROHCLz3pFhp2IL32O+Awt/f8F3OaOa+SNzWtHn6XW8rtsVrgWj8yki6AqntRs3fuOBTyUReJRLuyPjCfbVjGNKNljX7lf53WUWD/eAlQa6pAOjtz97DGi1VkTAQer4=
+  on:
+    branch: master
+    repo: swift-sunshine/swsui
+after_deploy:
+- "./.travis/travis-swscore.sh"
```

Comment:
I don't think you need to make this more complicated by introducing another separate script especially when that script is a very simple and short 2 lines long. In fact, you can shrink it to just 1 line and simply embed the curl call here - you get the same thing only without having to have this extra layer of script.

So this could be something like:

```
after_deploy:
- curl -s -X POST -H "Content-Type: application/json" -H "Accept: application/json" -H "Travis-API-Version: 3" -H "Authorization: token $TRAVIS_TOKEN" -d "{"request":{"message": "Triggered by swsui change: $TRAVIS_COMMIT", "branch":"$TRAVIS_BRANCH"}}" https://api.travis-ci.org/repo/swift-sunshine%2Fswscore/requests

```

Reply:
Done